m3/src/main/java/com/uber/m3/tally/m3/thrift/TUdpTransport.java

```diff
@@ -39,10 +39,10 @@
     //       which is set at 65,535 = 8 bytes (header) + 65,527 bytes (data)
     public static final int UDP_DATA_PAYLOAD_MAX_SIZE = 65527;
 
-    public final Object receiveLock = new Object();
-    public final Object sendLock = new Object();
+    protected final Object receiveLock = new Object();
```

Comment:
can we just keep this final? Wondering why do we need to expose it outside.

Reply:
sorry, i meant private final`

Reply:
Done